feature-group-definitions/fetch.yml

```diff
@@ -0,0 +1,89 @@
+spec: https://fetch.spec.whatwg.org/
+caniuse: fetch
+status:
+  baseline: false
+  support:
+    chrome: "105"
+    chrome_android: "105"
+    edge: "105"
+compat_features:
+  - api.Headers
+  - api.Headers.Headers
+  - api.Headers.append
+  - api.Headers.delete
+  - api.Headers.entries
+  - api.Headers.forEach
+  - api.Headers.get
+  # - api.Headers.getSetCookie # later addition
+  - api.Headers.has
+  - api.Headers.keys
+  # - api.Headers.lexicographical_sorting # unsupported in Safari, for a long time
```

Comment:
Dropping the feature from the computation simply because it's "unsupported" somewhere feels like cheating for no good reason. Also, "for a long time" suggests that this could be raised as an issue against the spec, either to drop the sorting, turn the MUST into a SHOULD, or to provide incentive to implement the feature correctly everywhere.

The comment could perhaps be something like "unsupported in Safari, non essential in common Fetch scenarios, see [url] for discussion".

Reply:
I don't think we disagree. I wouldn't necessarily change the approach compared to other features though. Developers trust and rely on fetch in browsers because it is widely supported for all practical usages of fetch. Which also means that we should be able to find good enough rationale to exclude subfeatures that are not widely supported...

For headers sorting, from an app perspective, I suspect the order really only matters for presentation purpose. That's definitely not in the critical path for using "Fetch".

From a spec perspective, one of the reasons why ordering might have been specified is that it is [good practice to specify ordering](https://w3c.github.io/fingerprinting-guidance/#specify-ordering)... I doubt lack of ordering in this case would reveal anything that wouldn't be trivial to get otherwise.